book/novel_applications/payment/private_payment.html

```diff
@@ -0,0 +1,449 @@
+                        link.setAttribute('tabIndex', sidebar === 'visible' ? 0 : -1);
+                    });
+                </script>
+
+                <div id="content" class="content">
+                    <main>
+                        <h1 id="private-payment"><a class="header" href="#private-payment">Private Payment</a></h1>
+<p><img src="../../assets/images/private_payment.png" alt="Private payment" /></p>
+<h2 id="the-why"><a class="header" href="#the-why">The Why?</a></h2>
+<h3 id="introduction"><a class="header" href="#introduction">Introduction</a></h3>
+<p>Efficient payments are a backbone of human societies. They enable exchange of value for goods and services, facilitate social functioning (participating in activities e.g., going to the restaurant), and support economic growth. Without them, commerce would grind to a halt, financial stability would be undermined, and access to essential services would be severely limited, potentially leading to economic stagnation and social disruption.</p>
+<h3 id="blockchains-as-a-payments-infrastructure"><a class="header" href="#blockchains-as-a-payments-infrastructure">Blockchains as a Payments Infrastructure</a></h3>
+<p>Since the very first blockchain, Bitcoin, payments have been the flagship use case. With further developments like Ethereum and alternative L1's (Polygon, Tron, Near) and innovations like rollups (Base, Starknet, Scroll), payments have been made faster, cheaper, and more powerful (with additional functionalities e.g., through smart contracts).</p>
+<div class="table-wrapper"><table><thead><tr><th>Blockchain</th><th>Bitcoin</th><th>Ethereum</th><th>Polygon</th><th>Base</th></tr></thead><tbody>
+<tr><td>Avg. tx fee ($)</td><td>0.68</td><td>1.85</td><td>0.003</td><td>0.00005</td></tr>
+<tr><td>Cost</td><td>High</td><td>High</td><td>Low</td><td>Virtually free</td></tr>
+<tr><td>Speed</td><td>Low</td><td>Low</td><td>High</td><td>High</td></tr>
+<tr><td>UX</td><td>Poor</td><td>Poor</td><td>Poor</td><td>Average</td></tr>
+<tr><td>Network</td><td>Decentralized</td><td>Decentralized</td><td>Less decentralized</td><td>Centralized</td></tr>
+<tr><td>Privacy</td><td>None</td><td>None</td><td>None</td><td>None</td></tr>
+</tbody></table>
+</div>
+<p>Nonetheless, for years blockchain payments have remained expensive and slow, especially at the base layers due to high gas fees and slow finality (~1 hour for Bitcoin and 12 minutes for Ethereum), complex to make due to bad UI/UX, and lack essential functionalities like privacy.</p>
+<p>This has prevented blockchains from acquiring the role of leading payment infrastructure and forced users to continue relying on classical centralized infrastructures (e.g., Visa, Mastercard, PayPal, Venmo, Revolut, Banks, etc.).</p>
+<h3 id="the-importance-of-privacy-in-payments"><a class="header" href="#the-importance-of-privacy-in-payments">The Importance of Privacy in Payments</a></h3>
```

Comment:
Here you said: 
> Instead of this paragraph I would say, we now see traction for payments because they are so cheap and fast on Tron, Celo, Bnb.
And
> I like this chapter. I would just shorten it a bit and merge this chapter "The importance of privacy in payments" with the first one.

I am not sure what to do